frontend/App.js

```diff
@@ -1,20 +1,11 @@
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import React from 'react';
+import Login from './screens/Login';
+import SignUpPage from './screens/SignUpPage';
+import Navigator from './routes/LogInNavigation';
```

Comment:
change name to importNavigator instead of Navigator

Reply:
why would i change this?